.gitignore

```diff
@@ -1,3 +1,5 @@
+.*un~
```

Comment:
Not unless we expect `.*~` to include stuff we want, but I tend to be conservative adding `.*` to .gitignore since I do this almost without thinking due to local vim undo files.